### DISCUSSION TITLE
PGO dont work

### DISCUSSION THREAD

**Discussion by dhn123257** [bevyengine/bevy#13405] [Q&A]
![20240517-113649](https://github.com/bevyengine/bevy/assets/68134343/a1edd9be-1def-449a-9b16-8a8f8434bdff)
how can i fix it?

**Comment by JMS55** [comment]
This is the github for Bevy Engine, not Unreal Engine. I think you have the wrong website?

**Reply by soqb** [reply]
:skull:
